$OpenBSD: patch-src_FDM_UIUCModel_uiuc_aircraft_h,v 1.1 2019/02/08 18:53:42 jasper Exp $

Fix build with libc++7: error: reference to 'stack' is ambiguous

Index: src/FDM/UIUCModel/uiuc_aircraft.h
--- src/FDM/UIUCModel/uiuc_aircraft.h.orig
+++ src/FDM/UIUCModel/uiuc_aircraft.h
@@ -150,7 +150,7 @@
 #include "uiuc_parsefile.h"
 #include "uiuc_flapdata.h"
 
-typedef stack :: iterator LIST;
+typedef ::stack :: iterator LIST;
 
 /* Add more keywords here if required*/
 enum {init_flag = 1000, geometry_flag, controlSurface_flag, controlsMixer_flag, 
